microapps/ArTabletopApp/app/src/main/java/com/arcgismaps/toolkit/artabletopapp/screens/MainScreen.kt

```diff
@@ -52,6 +70,8 @@ import com.arcgismaps.toolkit.ar.rememberTableTopSceneViewStatus
 import com.arcgismaps.toolkit.artabletopapp.R
 import kotlinx.coroutines.launch
 
+private const val ACCEPTED_PRIVACY_INFO = "ACCEPTED_PRIVACY_INFO"
```

Comment:
Naming suggestion to add a prefix "KEY_PREF_":
```suggestion
private const val KEY_PREF_ACCEPTED_PRIVACY_INFO = "ACCEPTED_PRIVACY_INFO"
```